### PR TITLE
Add roadmap for Q2

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

**Review by tpayet — APPROVED** [review]
looks good to me!

**Review by alallema — DISMISSED** [review]

**Review by bidoubiwa — APPROVED** [review]
LGTM !

File: roadmap/2021.md [hunk 1]
```diff
@@ -21,25 +21,54 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
...
+- [X] (_Still in progress during all year long_) Start improving our wrapper into SDKs by making more friendly search methods
+- (_Reported to Q2_) Dart SDK: normalize and integrate it to our batch of SDKs
+- (_Reported to Q2_) Go SDK refactoring
+- (_Reported to Q2_) Swift SDK error handler
+- (_Reported to Q2_) Dotnet SDK error handler
 
 Front-End:
 - [X] Transform to TypeScript InstantMeiliSearch
-- [ ] Add tests and more documentation to InstantMeiliSearch
+- [ ] (_Reported to Q2_) Add tests and more documentation to InstantMeiliSearch
```
Comment (curquiza) [inline roadmap/2021.md:35]:
```suggestion
- (_Reported to Q2_) Add tests and more documentation to InstantMeiliSearch
```

**Comment by curquiza** [comment]
Thanks for your time everyone!

**Pull request merged**